ClosedXML/XLHelper.cs

```diff
@@ -21,8 +21,12 @@ public static class XLHelper
 
         private const Int32 TwoT26 = 26 * 26;
 
-        internal static readonly Graphics Graphic = Graphics.FromImage(new Bitmap(200, 200));
-        internal static readonly Double DpiX = Graphic.DpiX;
+        private static readonly Lazy<Graphics> graphics = new Lazy<Graphics>(() => Graphics.FromImage(new Bitmap(200, 200)));
+        internal static Graphics Graphic { get => graphics.Value; }
+
+        private static readonly Lazy<Double> dpiX = new Lazy<double>(() => Graphic.DpiX);
```

Comment:
This field is not necessary. If you change the internal property to 
`internal static Double DpiX { get => Graphic.DpiX; }`
it'll be lazy-loaded already.